signup-service/src/main/java/io/mosip/signup/helper/NotificationHelper.java

```diff
@@ -33,11 +34,14 @@ public class NotificationHelper {
     @Value("${mosip.signup.send-notification.endpoint}")
     private String sendNotificationEndpoint;
 
+    @Value("#{${mosip.signup.supported-languages}}")
+    private List<String> supportedLanguages;
+
     public RestResponseWrapper<NotificationResponse> sendSMSNotification
             (String number, String locale, String templateKey, Map<String, String> params){
 
-        locale = locale != null ? locale : "khm";
-        String message = locale.equalsIgnoreCase("eng") ?
+        locale = locale != null ? locale : supportedLanguages.get(0);
+        String message = locale.equalsIgnoreCase(supportedLanguages.get(1)) ?
```

Comment:
We should not assume that the secondary language notification template is not encoded always. Can we take this from properties?
something like this? mosip.signup.sms-notification-template.forgot-password.encoded.khm